detection_rules/schemas/definitions.py

```diff
@@ -81,6 +81,8 @@
 TimelineTemplateId = NewType('TimelineTemplateId', str, validate=validate.OneOf(list(TIMELINE_TEMPLATES)))
 TimelineTemplateTitle = NewType('TimelineTemplateTitle', str, validate=validate.OneOf(TIMELINE_TEMPLATES.values()))
 UUIDString = NewType('UUIDString', str, validate=validate.Regexp(UUID_PATTERN))
+NewTermsFields = NewType('NewTermsFields', List[NonEmptyStr], validate=validate.Length(min=0, max=1))
```

Comment:
For now, this field only accepts one object but is being worked on to accept multiple. We will need to revisit the validation length after this is solved.

Reference - https://github.com/elastic/kibana/issues/142862#issuecomment-1279025890

Reply:
Added. However I may need to revisit the PR to Kibana that allows this rule type to exist. When I created a New Terms rule from the Kibana UI, the option to add a query still existed. If this type of rule show not have a query then that will be good to know and we can adjust in our code for this during validation or with unit testing.